app/src/main/java/com/example/playlistmakernewversion/domain/OnPlayerStateChangeListener.kt

```diff
@@ -0,0 +1,6 @@
+package com.example.playlistmakernewversion.domain
+
+interface OnPlayerStateChangeListener {
```

Comment:
интерфейс можно пометить функциональным:
```suggestion
fun interface OnPlayerStateChangeListener {
```
Чтобы не приходилось везде писать `object : OnPlayerStateChangeListener`, а сразу написать лямбду:
`OnPlayerStateChangeListener  {}`